File: 26.12.2025/5.java
import java.util.Scanner;

class VotingSystem {
    static int votesA = 0, votesB = 0;

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int choice;

        do {
            System.out.println("\n1.Vote Candidate A 2.Vote Candidate B 3.Results 4.Exit");
            choice = sc.nextInt();

            switch (choice) {
                case 1 -> votesA++;
                case 2 -> votesB++;
                case 3 -> {
                    System.out.println("Candidate A Votes: " + votesA);
                    System.out.println("Candidate B Votes: " + votesB);
                }
            }
        } while (choice != 4);
    }
}

File: 28.12.2025/5.py
votes_a = votes_b = 0

while True:
    print("\n1.Vote A 2.Vote B 3.Result 4.Exit")
    ch = int(input("Enter choice: "))

    if ch == 1:
        votes_a += 1
    elif ch == 2:
        votes_b += 1
    elif ch == 3:
        print("Votes A:", votes_a)
        print("Votes B:", votes_b)
    elif ch == 4:
        break
